virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -22,6 +22,11 @@ When you create virtual machines from the web console, select a virtual machine
 
 Templates without a boot source are labeled as *Boot source required*. You can use these templates if you complete the steps for xref:../../virt/vm_templates/virt-creating-vm-template.adoc#virt-adding-a-boot-source-web_virt-creating-vm-template[adding a boot source to the virtual machine].
 
+[IMPORTANT]
+====
+Due to differences in storage behavior, some virtual machine templates are incompatible with `SingleReplica` mode. You must remove the `evictionStrategy` from any templates or virtual machines that use `DataVolumes` and storage profiles to ensure compatibility with `SingleReplica` mode.
```

Comment:
Can you please rewrite this text to match the last bullet list item in the `virt-single-node-cluster` module? Same comment applies to the note in the `virt-about-vm-templates.adoc` module as well.